src/spikeinterface/postprocessing/tests/test_align_sorting.py

```diff
@@ -12,29 +7,44 @@
 
 
 def test_align_sorting():
+    """
+    `align_sorting()` shifts, in time, the spikes belonging to a unit.
+    For each unit, an offset is provided and the spike peak index is shifted.
+
+    This test creates a sorting object, then creates an 'unaligned' sorting
+    object in which the peaks for some of the units are shifted. Next, the `align_sorting()`
+    function is unused to unshift them, and the original sorting spike train
+    peak times compared with the corrected sorting train.
+    """
     sorting = generate_sorting(durations=[10.0], seed=0)
-    print(sorting)
 
     unit_ids = sorting.unit_ids
 
     unit_peak_shifts = {unit_id: 0 for unit_id in unit_ids}
     unit_peak_shifts[unit_ids[-1]] = 5
     unit_peak_shifts[unit_ids[-2]] = -5
 
-    # sorting to dict
-    d = {unit_id: sorting.get_unit_spike_train(unit_id) + unit_peak_shifts[unit_id] for unit_id in sorting.unit_ids}
-    sorting_unaligned = NumpySorting.from_unit_dict(d, sampling_frequency=sorting.get_sampling_frequency())
-    print(sorting_unaligned)
+    shifted_unit_dict = {
+        unit_id: sorting.get_unit_spike_train(unit_id) + unit_peak_shifts[unit_id] for unit_id in sorting.unit_ids
+    }
+    sorting_unaligned = NumpySorting.from_unit_dict(
+        shifted_unit_dict, sampling_frequency=sorting.get_sampling_frequency()
+    )
 
     sorting_aligned = align_sorting(sorting_unaligned, unit_peak_shifts)
-    print(sorting_aligned)
-
-    for start_frame, end_frame in [(None, None), (10000, 50000)]:
```

Comment:
the start/end being not none was here to tests corner cases with the shift.
for removing then ?

Reply:
I think these values [were not currently used](https://github.com/SpikeInterface/spikeinterface/blob/17ee78552ddff147e1aaefcac70364c4a8520d99/src/spikeinterface/postprocessing/tests/test_align_sorting.py#L32) so removed, but definitely happy to add these back to extend the tests. Their use should be to slice both spike-trains before testing equality?